src/WalkingModule/src/Module.cpp

```diff
@@ -577,7 +577,7 @@ bool WalkingModule::updateModule()
         }
 
         // get feedbacks and evaluate useful quantities
-        if(!m_robotControlHelper->getFeedbacks(10))
+        if(!m_robotControlHelper->getFeedbacks(100))
```

Comment:
I changed the maximum attempts number, since in simulation of Gazebo it seems it needs more time to receiving the base data! Does it cause some problems in the experiment?

Reply:
I think it is fine. We are already moving to that direction in #41 (see https://github.com/robotology/walking-controllers/blob/ca8128098d3e8d6ba7f45ac3e22bf89f871e270b/src/WalkingModule/src/Module.cpp#L501). Anyways, it is a bit weird you get that delay in simulation.